pages/api/prospect/get-by-id.tsx

```diff
@@ -0,0 +1,9 @@
+import prisma from '../../../lib/prisma'
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
let's have a more general function named get which receives the parameter as url parameters or as json. Also, maybe we can use the same method to get a list of prospects